### DISCUSSION TITLE
Jameson Humane Courtesy Post Application page not working

### DISCUSSION THREAD

**Discussion by AJJasper** [sheltermanager/asm3#1484] [Q&A / Help]
UPDATE - I was finally able to get the form to submit - all 4 times - even with glitches of the same error page in between. The page should still be checked for security connection issues. 

Hello - I am trying to post info about my friend's four animals to the Jameson Ranch website on the form at this address: 

https://us12b.sheltermanager.com/service?account=az1802&method=online_form_html&formid=48

When I try to submit the info, I get back a message:

"Secure Connection Failed
An error occurred during a connection to us12b.sheltermanager.com.

The page you are trying to view cannot be shown because the authenticity of the received data could not be verified.
    Please contact the website owners to inform them of this problem."
    
I am on a Macbook Air OS Monterey 12.7.5 and using Mozilla Firefox.

This is a time-sensitive task to get these animals rehomed and was hoping to get some help getting the word out through Jameson.  Thank you!

**Comment by Tetzor** [comment]
Hi, an SSL error with our site would prevent the form from actually loading at all. As the error occurs after the forms are successfully posted, the problem must be with the site they have chosen to redirect to after submission of the form  (the Jameson Ranch website). You could try a different browser like Safari to see if that solves the problem. I did try to recreate it using FireFox, but I use Linux rather than a Mac and  the error didn't occur for me.

**Comment by AJJasper** [comment]
Thank you for getting back to me! I was able to get the form to work, even though it would give me the same error message a handful of times each time I’d submit it. But by the second or third time would go through.

I’ve also let Jameson Ranch know about the issue, and that makes sense - it would land back on their website.

Thank you again!

Sent from my iPhone

On Jun 18, 2024, at 4:02 AM, Jon ***@***.***> wrote:

﻿

Hi, an SSL error with our site would prevent the form from actually loading at all. As the error occurs after the forms are successfully posted, the problem must be with the site they have chosen to redirect to after submission of the form (the Jameson Ranch website). You could try a different browser like Safari to see if that solves the problem. I did try to recreate it using FireFox, but I use Linux rather than a Mac and the error didn't occur for me.
